packages/plexus/README.md

```diff
@@ -1,13 +1,914 @@
 # plexus
 
-plexus is a React component for rendering directed graphs.
+A React component for directed graphs.
 
-## Why?
+![NPM @jaegertracing/plexus](https://img.shields.io/npm/v/@jaegertracing/plexus?color=08979c&style=for-the-badge&labelColor=444) ![Apache 2.0 License](https://img.shields.io/npm/l/@jaegertracing/plexus?color=096dd9&style=for-the-badge&labelColor=444&label=)
 
-To support directed graphs in Jaeger-UI, we surveyed the JavaScript libraries and utilities available for rendering directed graphs. The landscape is impressive, but we concluded the venerable [GraphViz](https://graphviz.gitlab.io/) ([alt](https://www.graphviz.org/)) is the right tool for us.
+<!-- pro-tip: Generate the TOC here at https://magnetikonline.github.io/markdown-toc-generate/ -->
 
-GraphViz is awesome, but the output formats don't fit our needs. We've elected to use GraphViz to generate the layouts (node positioning, edge routing) and React for rendering.
+- [About](#about)
```

Comment:
I don't know about the browser compatibility or how it relates to `About`. Can you elaborate4?

Reply:
My bad, I meant "this" as the table of contents / document, not the `About` section.
I don't know the browser compatibility, but given `totalMemory` limitation I was wondering if there was any known browser compatibility issues? If not, this can be disregarded.

Reply:
The `totalMemory` config has to do with how much memory is pre-allocated by GraphViz (IIUC), not related to browser versions or anything.